airmozilla/main/static/main/js/eventstatus.js

```diff
@@ -0,0 +1,22 @@
+$(function () {
+    var TEN_SECONDS = 10000;
+    var url = location.pathname + 'status/';
+    var currentEventStatus = $('.islive').data('event-status');
+
+    var interval = setInterval(function () {
+        refreshIfStatusChanges();
+    }, TEN_SECONDS);
+
+    function refreshIfStatusChanges() {
+        $.get(url)
+            .then(function (data) {
+                if (currentEventStatus !== data.status) {
+                    location.reload();
+                }
+            })
+            .fail(function () {
+                console.error("Failed to fetch event status. Stops looking.");
+                clearInterval(interval);
+            });
+    }
+});
```

Comment:
You're definitely on the right track but consider refactoring here a bit. Everything that is only ever used once, don't bother defining it first. 
For the example, the `url`. And the `refreshIfStatusChanges` function.